legacy/ExportPIM.js

```diff
@@ -11,12 +12,18 @@ async function LegacyExportPIM(req) {
   if (reqBody.recordIds.length == 0) {
     return 'Error';
   }
-  let recordsAndCols;
+  let daDownloadDetailsList, recordsAndCols;
   try {
-    recordsAndCols = await PimStructure(reqBody, isListPageExport);
+    ({daDownloadDetailsList, recordsAndCols} = await PimStructure(reqBody, isListPageExport));
   } catch (err) {
     console.log('error: ', err);
   }
+
+  const baseFileName = createBaseFileName();
+  const filename = 'Product-Export_' + baseFileName + '.csv';
```

Comment:
tho this works as expected I think the in favor javascript method of concatination is 
```
const filename = `Product-Export_${baseFileName}.csv`
```